requirements.txt

```diff
@@ -1,6 +1,6 @@
 django>=2.2,<3.2
 django-model-utils>=4.0
 jsonfield>=3.1.0,<4.0.0
-cryptography>=2.4.0,<3.1.0
+cryptography>=2.5.0,<3.2.0
```

Comment:
@DaffyTheDuck since we are updating to version 3.1.1, why don't you write this as 
`cryptography>=3.1.1,<3.2.0`

Reply:
Sure!